estate/views/estate_menus.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<odoo>
+    <menuitem id="estate_menu_root" name="Estate">
+        <menuitem id="estate_menu_advertisement" name="Advertisement">
+            <menuitem id="estate_propert_action_action" action="estate_property_model_action"/>
```

Comment:
```suggestion
            <menuitem id="estate_property_action_action" action="estate_property_model_action"/>
```
You probably forgot a 'y'

Reply:
I figured that when i was doing my chapter 7 :)